ejb/WFLY-6892-ejb-access-logging.adoc

```diff
@@ -0,0 +1,387 @@
+
+
+=== New elements in wildfly-ejb3_6_0.xsd schema
+ejb30 subsystem is extended with a new top-level element access-logging, defined as follows:
+[source,xml]
+----
+    <xs:element name="access-logging" type="accessLoggingType" minOccurs="0" maxOccurs="1" />
+
+    <xs:complexType name="accessLoggingType">
+            <xs:choice minOccurs="0" maxOccurs="unbounded">
+                <xs:element name="console-access-log" type="consoleAccessLogType"/>
+                <xs:element name="file-access-log" type="fileAccessLogType"/>
+                <xs:element name="periodic-rotating-file-access-log" type="periodicFileAccessLogType"/>
+                <xs:element name="size-rotating-file-access-log" type="sizeFileAccessLogType"/>
+                <xs:element name="server-log-access-log" type="serverLogAccessLogType"/>
+                <xs:element name="formatter" type="formatterType"/>
+            </xs:choice>
+        </xs:complexType>
+
+        <xs:complexType name="consoleAccessLogType">
+            <xs:annotation>
+                <xs:documentation>
+                    Defines an access log which writes to the console.
+                </xs:documentation>
+            </xs:annotation>
+            <xs:all>
+                <xs:element name="encoding" type="valueType" minOccurs="0"/>
+                <xs:element name="formatter" type="accessLogFormatterType" minOccurs="0"/>
+                <xs:element name="target" minOccurs="0">
+                    <xs:complexType>
+                        <xs:attribute name="name" use="required">
+                            <xs:simpleType>
+                                <xs:restriction base="xs:token">
+                                    <xs:enumeration value="System.out"/>
+                                    <xs:enumeration value="System.err"/>
+                                    <xs:enumeration value="console"/>
+                                </xs:restriction>
+                            </xs:simpleType>
+                        </xs:attribute>
+                    </xs:complexType>
+                </xs:element>
+            </xs:all>
+            <xs:attribute name="autoflush" type="xs:boolean" use="optional" default="true"/>
+            <xs:attribute name="name" type="xs:string" use="required"/>
+            <xs:attribute name="enabled" type="xs:boolean" use="optional" default="true"/>
+        </xs:complexType>
+
+        <xs:complexType name="serverLogAccessLogType">
+            <xs:annotation>
+                <xs:documentation>
+                    Defines an access log which writes to the server log.
+                </xs:documentation>
+            </xs:annotation>
+            <xs:all>
+                <xs:element name="formatter" type="accessLogFormatterType" minOccurs="0"/>
+            </xs:all>
+            <xs:attribute name="name" type="xs:string" use="required"/>
+            <xs:attribute name="enabled" type="xs:boolean" use="optional" default="true"/>
```

Comment:
Same comment as above for this.

Reply:
Makes sense, and I'll remove this attribute (enabled).